.github/workflows/pr.yml

```diff
@@ -25,8 +25,9 @@ jobs:
         run: go mod download
       - name: Build Binaries
         run: |
-          make cross
-          make e2e-cross
+          sudo apt-get update
```

Comment:
lets remove irellevelent changes, and lets remove the changes from both master and pr 
untill we make the PR and Mater yamls generated by a template